src/azul/__init__.py

```diff
@@ -528,19 +529,16 @@ def __attrs_post_init__(self):
         def is_integration_test_catalog(self) -> bool:
             return self._it_catalog_re.match(self.name) is not None
 
-        @property
-        def is_internal(self):
-            # FIXME: Remove the second term
-            #        https://github.com/DataBiosphere/azul/issues/2865
-            return self.is_integration_test_catalog or self.name == 'dcp3'
-
         @classmethod
         def from_json(cls, name: str, spec: JSON) -> 'Config.Catalog':
             plugins = {
```

Comment:
This is more feedback for the parent PR, but I wanted to point out that there's no enforcement for *which* plugins are configured. In other words, `plugins = {}` would parse here as well as some made up plugins. We might want to be stricter about that.

Reply:
I'll consider adding something to the parent PR.

Reply:
https://github.com/DataBiosphere/azul/pull/3109/files#diff-a5ab528d23a14c23dd1439c679fe8f2af5e5bcab793aee95451aeefe4f7b86dfR521

Reply:
```
config.Catalog.from_json(name='hi', spec=dict(atlas='foo', plugins={}))
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/Users/amar/dev/hca/azul/src/azul/__init__.py", line 543, in from_json
    return cls(name=name, atlas=spec['atlas'], plugins=plugins)
  File "<attrs generated init azul.Config.Catalog>", line 6, in __init__
    self.__attrs_post_init__()
  File "/Users/amar/dev/hca/azul/src/azul/__init__.py", line 523, in __attrs_post_init__
    require(all_types == configured_types,
  File "/Users/amar/dev/hca/azul/src/azul/__init__.py", line 1097, in require
    reject(not condition, *args, exception=exception)
  File "/Users/amar/dev/hca/azul/src/azul/__init__.py", line 1112, in reject
    raise exception(*args)
azul.RequirementError: ('Missing or extra plugin types', {'repository', 'metadata'})
```